package.json

```diff
@@ -21,21 +22,29 @@
     "babel-jest": "^24.9.0",
     "core-js": "^2.6.9",
     "eslint": "^6.5.0",
+    "eslint-plugin-react": "^7.14.3",
     "eslint-plugin-vue": "^5.2.3",
     "flatted": "^2.0.1",
     "husky": "^3.0.7",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.9.0",
     "jest-serializer-vue": "^2.0.2",
     "lerna": "^3.16.4",
+    "octicons-react": "^0.18.9",
     "octicons-vue": "^0.18.9",
     "prismjs": "^1.17.1",
+    "prop-types": "^15.7.2",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
+    "react-router-dom": "^5.1.0",
+    "react-transition-group": "^4.3.0",
     "rimraf": "^3.0.0",
     "rollup": "^1.22.0",
     "rollup-plugin-buble": "^0.19.8",
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-postcss": "^2.0.3",
+    "rollup-plugin-scss": "^1.0.2",
```

Comment:
We actually don't need `rollup-plugin-scss` because the styling of transitions is injected into outputs.